var/spack/repos/builtin/packages/adol-c/package.py

```diff
@@ -25,39 +25,53 @@
 from spack import *
 
 
-class AdolC(Package):
+class AdolC(AutotoolsPackage):
     """A package for the automatic differentiation of first and higher
     derivatives of vector functions in C and C++ programs by operator
     overloading."""
     homepage = "https://projects.coin-or.org/ADOL-C"
     url      = "http://www.coin-or.org/download/source/ADOL-C/ADOL-C-2.6.1.tgz"
 
-    version('develop',  svn='https://projects.coin-or.org/svn/ADOL-C/trunk/')
+    version('develop',  git='https://gitlab.com/adol-c/adol-c.git',
+            branch='master')
     version('2.6.2', '0f9547584c99c0673e4f81cf64e8d865')
     version('2.6.1', '1032b28427d6e399af4610e78c0f087b')
 
     variant('advanced_branching', default=False,
             description='Enable advanced branching to reduce retaping')
+    variant('atrig_erf', default=True,
+            description='Enable arc-trig and error functions')
     variant('doc',      default=True,  description='Install documentation')
     variant('openmp',   default=False, description='Enable OpenMP support')
     variant('sparse',   default=False, description='Enable sparse drivers')
     variant('tests',    default=True,
             description='Build all included examples as a test case')
 
+    # Build dependencies
+    depends_on('automake', type='build', when='@develop')
+    depends_on('autoconf', type='build', when='@develop')
+    depends_on('libtool',  type='build', when='@develop')
```

Comment:
It is generally needed to build configure. In fact, it is so commonly needed that we added checks to make sure that it is always present when running `autoreconf`:
https://github.com/LLNL/spack/blob/develop/lib/spack/spack/build_systems/autotools.py#L207
@alalazo maybe these aren't working?